cypress/e2e/API/api_EngageSphere.cy.js

```diff
@@ -0,0 +1,142 @@
+                expect(response.body.pageInfo.totalPages).to.be.eq(1)
+                response.body.customers.forEach((customer) => {
+                    expect(customer).to.have.property('size').to.be.eq('Medium');
+                    expect(customer.employees, `Customer ${customer.name} has ${customer.employees} employees, so the company size is ${customer.size}`)
+                        .to.be.at.greaterThan(99).and.to.be.lessThan(1000)
+                });
+
+            });
+        });
+
+        it('Filters Enterprise size customers', () => {
+            cy.request('GET', 'http://localhost:3001/customers?size=Enterprise&limit=50').then((response) => {
+                expect(response.status).to.eq(200);
+                expect(response.body.pageInfo.totalCustomers).to.be.eq(13)
+                expect(response.body.pageInfo.totalPages).to.be.eq(1)
+                response.body.customers.forEach((customer) => {
+                    expect(customer).to.have.property('size').to.be.eq('Enterprise');
+                    expect(customer.employees, `Customer ${customer.name} has ${customer.employees} employees, so the company size is ${customer.size}`)
+                        .to.be.at.greaterThan(999).and.to.be.lessThan(10000)
+                });
+
+
+            });
+        });
+
+        it('Filters Large Enterprise size customers', () => {
+            cy.request('GET', 'http://localhost:3001/customers?size=Large%20Enterprise&limit=50').then((response) => {
+                expect(response.status).to.eq(200);
+                expect(response.body.pageInfo.totalCustomers).to.be.eq(18)
+                expect(response.body.pageInfo.totalPages).to.be.eq(1)
+                response.body.customers.forEach((customer) => {
+                    expect(customer).to.have.property('size').to.be.eq('Large Enterprise');
+                    expect(customer.employees, `Customer ${customer.name} has ${customer.employees} employees, so the company size is ${customer.size}`)
+                        .to.be.at.greaterThan(9999).and.to.be.lessThan(50000)
+                });
+
+
+            });
+        });
+
+        it('Filters Very Large Enterprise size customers', () => {
+            cy.request('GET', 'http://localhost:3001/customers?limit=50&size=Very%20Large%20Enterprise').then((response) => {
+                expect(response.status).to.eq(200);
+                expect(response.body.pageInfo.totalCustomers).to.be.eq(10)
+                expect(response.body.pageInfo.totalPages).to.be.eq(1)
+                response.body.customers.forEach((customer) => {
+                    expect(customer).to.have.property('size').to.be.eq('Very Large Enterprise');
+                    expect(customer.employees, `Customer ${customer.name} has ${customer.employees} employees, so the company size is ${customer.size}`)
+                        .to.be.at.greaterThan(49999)
+                });
+            });
+        });
+    })
+})
```

Comment:
É uma boa prática (e convenção utilizada pela comunidade JS) deixar uma linha em branco ao final de cada arquivo.

```suggestion
})

```